RELEASES.md

```diff
@@ -1,3 +1,151 @@
+Version 1.33.0 (2019-02-28)
+==========================
+
+Language
+--------
+- [You can now use the `cfg(target_vendor)` attribute.][57465] E.g.
+  `#[cfg(target_vendor="linux")] fn main() { println!("Hello Linux!"); }`
```

Comment:
I don't think this example makes sense. Maybe use one of the vendors shown in the [reference](https://doc.rust-lang.org/reference/conditional-compilation.html#target_vendor)?

Reply:
cc @Mark-Simulacrum -- presumably we don't want to change this now because we already promoted beta=>stable... but we could merge it into nightly for posterity?

Reply:
PRs are welcome!